signature/src/suite/ed25519_2020/ed25519_2020_error.rs

```diff
@@ -0,0 +1,11 @@
+#[derive(Debug, thiserror::Error)]
```

Comment:
Might be better to have  signature/src/suite/ed25519_2020/error.rs instead of signature/src/suite/ed25519_2020/ed25519_2020_error.rs because it is a part of ed25519_2020 mod already

Reply:
Okay sounds good, I had renamed it to something else to make searching easier but I'm having to change it to `error.rs`